fastapi/security/ir_rule+acl.xml

```diff
@@ -39,7 +39,7 @@
         <field name="model_id" ref="base.model_res_partner" />
         <field
             name="domain_force"
-        > ['|', ('user_id', '=', user.id), ('id', '=', authenticated_partner_id)]</field>
+        > ['|', ('user_ids', '=', user.id), ('id', '=', authenticated_partner_id)]</field>
```

Comment:
@Tisho99 wouldn't it be more accurate to write `('user_ids', 'in', user.id)` ?

Reply:
@lmignon 

Everything works with the Odoo ORM because they've designed their operators that way, but from what I know, the most commonly used nomenclature is the equals operator and the following: ('user_ids', 'in', [user.id]).

I think the most readable way would be to use the "like" operator, but I think it's also slower.